index.d.ts

```diff
@@ -32,6 +41,44 @@ declare namespace elementReady {
 		*/
 		stop(): void;
 	}
+
+  /**
+  Called on each new element that matches the selector in the DOM.
+  */
+  type SubscribeCallback<T> = (el: T) => any;
```

Comment:
This type is incorrect. According to the JS code, the return value is not used.
```suggestion
  type SubscribeCallback<T> = (element: T) => any;
```